projects/plugins/wpcomsh/composer.json

```
@@ -134,10 +134,7 @@
 		"autorelease": true,
 		"autotagger": true,
```

Since this is part of the auto-tagger I'd probably re-work the config to look more like this:
```suggestion
		"autotagger": {
			"rolling-release": true
		},
```

---

Yep, added that, plus the `titlefmt` property.